receptorctl/pyproject.toml

```diff
@@ -1,6 +1,51 @@
+[project]
+name = "receptorctl"
+authors = [{name = "Red Hat", email = "info@ansible.com"}]
+description = "Receptorctl is a front-end CLI and importable Python library that interacts with Receptor over its control socket interface."
+readme = "README.md"
+dynamic = ["version"]
+dependencies = [
+  "python-dateutil==2.9.0",
+  "click==8.1.8",
+  "PyYAML==6.0.2",
```

Comment:
Pin to exact versions will cause conflicts, especially because the same receptor version is used in multiple AAP releases with different dependent rpm versions. Can you set lower and/or upper version constraint instead?